sdk/include/dxsdk/axextend.idl

```diff
@@ -554,21 +559,6 @@ interface IFileSinkFilter2 : IFileSinkFilter
 
 typedef IFileSinkFilter2 *PFILESINKFILTER2;
 
-[
```

Comment:
This file isn't in sync with Wine, that's why you're having trouble with it. Basically, you want to not delete things, only add.
I've commented below with the two that will actually fix compilation, but you should also keep all the other stuff.

Reply:
But if there are identical definitions on the same lines, but with different values, e. g. different uuids on the same line, should I also keep old uuid or can replace it with new one? :thinking: 
E. g., this:
https://github.com/reactos/reactos/blob/29c1089983ce678430a1c120a38b1a183aebcbbd/sdk/include/dxsdk/axextend.idl#L1153
In Wine it seems to be https://source.winehq.org/git/wine.git/blob/HEAD:/include/axextend.idl#l1194.

Reply:
The diffs don't display this very well, they'll line up the `[` and `]` lines. Look at things in terms of whole declarations instead. I.e. down where your example is, you want the file to have both the entire `ISeekingPassThru` and the entire `IMediaPropertyBag` declaration.

So you'll want:
```
[
    object,
    uuid(36b73883-c2c8-11cf-8b46-00805f6cef60),
    pointer_default(unique)
]
interface ISeekingPassThru : IUnknown
{
    HRESULT Init( [in] BOOL bSupportRendering, [in] IPin *pPin);
}


[
    local,
    object,
    uuid(6025a880-c0d5-11d0-bd4e-00a0c911ce86),
    pointer_default(unique)
]
interface IMediaPropertyBag : IPropertyBag
{
    typedef IMediaPropertyBag *LPMEDIAPROPERTYBAG;

    HRESULT EnumProperty(
            [in]  ULONG iProperty,
            [in, out] VARIANT *pvarPropertyName,
            [in, out] VARIANT *pvarPropertyValue);
}
```

Reply:
Okay. Thanks.
Although I already found out this interconnection recently.

Reply:
Fixed.
After proper fixing our axextend.idl, compilation succeeds again, even with enabled bdatif.idl and tuner.idl headers! :smiley: So that disabling was wrong, as I see. It is neccesarily to do it in a separate PR then @ThFabba? :thinking: